apps/api/src/domain/models/context/commands/connect-resources-with-note/connect-resources-with-note.command.spec.ts

```diff
@@ -0,0 +1,582 @@
+                                        toMemberCompositeIdentifier: toMember.compositeIdentifier,
+                                        toMemberContext: toMember.context,
+                                        fromMemberCompositeIdentifier:
+                                            fromMember.compositeIdentifier,
+                                        // overwrite the from member context with an inconsistent state
+                                        fromMemberContext: contextModelMap.get(
+                                            fromMember.context.type
+                                        ),
+                                    }),
+                                initialState: initialStateWithAllResourcesButNoConnections,
+                            });
+                        });
+                    });
+                });
+        });
+
+        describe(`when the resource type is a non-resource aggregate type on the property`, () => {
+            ['toCompositeIdentifier', 'fromCompositeIdentifier'].forEach((propertyName) =>
+                describe(propertyName, () => {
+                    Object.values(AggregateType)
+                        .filter((t) => !isResourceType(t))
+                        .forEach((invalidResourceType) => {
+                            describe(`when the resource type is: ${invalidResourceType}`, () => {
+                                it(`should fail with a type error`, async () => {
+                                    await assertCommandFailsDueToTypeError(
+                                        assertionHelperDependencies,
+                                        {
+                                            propertyName: propertyName,
+                                            invalidValue: {
+                                                type: invalidResourceType,
+                                                id: buildDummyUuid(567),
+                                            },
+                                        },
+                                        commandFsaFactory.build(buildDummyUuid(662))
+                                    );
+                                });
+                            });
+                        });
+                })
+            );
+        });
+
+        describe(`when the context for the a member has an unregistered context type`, () => {
+            const bogusContextType = 'totally-booogus8';
+
+            ['toMemberContext', 'fromMemberContext'].forEach((propertyName) => {
+                describe(`${propertyName} : ${bogusContextType}`, () => {
+                    it(`should fail with the expected error`, async () => {
+                        await assertCreateCommandError(assertionHelperDependencies, {
+                            systemUserId: dummySystemUserId,
+                            initialState: initialStateWithAllResourcesButNoConnections,
```

Comment:
Stick with the standard order of props here.

Reply:
It seems like this actually is the order that makes sense. Given that there is state X, build an FSA and attempt a command. Then check the result.